2.4/s2i/bin/assemble

```diff
@@ -9,9 +9,9 @@ echo "---> Enabling s2i support in httpd24 image"
 config_s2i
 
 echo "---> Installing application source"
-cp -Rf /tmp/src/. ./
-
-process_extending_files ${HTTPD_APP_ROOT}/src/httpd-post-assemble/ ${HTTPD_CONTAINER_SCRIPTS_PATH}/post-assemble/
+cp -aRf /tmp/src/. ./
```

Comment:
The change is needed here, since the permissions will be reset shortly after anyway. Though I do not think it would break anything if it would get added so I would be ok with it.

However, since the `-R` is superfluous as it is set by `-a` please either remove the `-R` or the `-a`.